admin_manual/configuration_server/config_sample_php_parameters.rst

```diff
@@ -607,7 +607,7 @@ available.
 
 ::
 
-	'updater.server.url' => 'https://updates.nextcloud.org/server/',
+	'updater.server.url' => 'https://updates.nextcloud.org/updater_server/',
```

Comment:
This will be overwritten by https://github.com/nextcloud/server/blob/master/config/config.sample.php#L508 (also the change directly above.